bundles/AdminBundle/Resources/public/js/pimcore/object/tags/advancedManyToManyObjectRelation.js

```diff
@@ -190,9 +190,9 @@ pimcore.object.tags.advancedManyToManyObjectRelation = Class.create(pimcore.obje
                     }
                 };
 
-                listeners = {
-                    "mousedown": this.cellMousedown.bind(this, this.fieldConfig.columns[i].key, this.fieldConfig.columns[i].type)
-                };
+                listeners = [
```

Comment:
does that change to an array of objects really work? 
docs say, listeners should be an object (not an array of objects). should we stick to documented ways of defining listeners?

Reply:
strange but it works! reverted as per your suggestion :-)